src/test/java/co/rsk/bitcoinj/script/ScriptTest.java

```diff
@@ -118,8 +100,8 @@ public void testScriptPubKey() {
         byte[] pubkeyBytes = Hex.decode(pubkeyProg);
         Script pubkey = new Script(pubkeyBytes);
         assertEquals("DUP HASH160 PUSHDATA(20)[33e81a941e64cda12c6a299ed322ddbdd03f8d0e] EQUALVERIFY CHECKSIG", pubkey.toString());
-        Address toAddr = new Address(PARAMS, pubkey.getPubKeyHash());
-        assertEquals("mkFQohBpy2HDXrCwyMrYL5RtfrmeiuuPY2", toAddr.toString());
+        Address toAddr = new Address(mainnetParams, pubkey.getPubKeyHash());
+        assertEquals("15jTWe6r9zqxkjjLFntAWADZosAwiuw4U5", toAddr.toString());
```

Comment:
we might use an `expectedConstant` to avoid magic strings in the code. WDYT?

Reply:
Updated.